nixos/doc/manual/man-nixos-rebuild.xml

```diff
@@ -24,10 +24,18 @@
       <arg choice='plain'><option>build</option></arg>
       <arg choice='plain'><option>dry-build</option></arg>
       <arg choice='plain'><option>dry-activate</option></arg>
+      <arg choice='plain'><option>build-ova</option></arg>
       <arg choice='plain'><option>build-vm</option></arg>
       <arg choice='plain'><option>build-vm-with-bootloader</option></arg>
     </group>
     <sbr />
+    <arg>
+      <group choice='opt'>
+        <arg choice='plain'><option>--configuration</option></arg>
```

Comment:
Why do we need that if you can do `NIXOS_PATH=nixos-config=/path/to/config nixos-rebuild build-vm`?